operator/apis/lifecycle/v1alpha2/common/phases.go

```diff
@@ -24,6 +24,14 @@ var phases = []KeptnPhaseType{
 	PhaseAppPreEvaluation,
 	PhaseAppPostEvaluation,
 	PhaseAppDeployment,
+	PhaseReconcileEvaluation,
+	PhaseReconcileTask,
+	PhaseCreateEvaluation,
+	PhaseCreateTask,
+	PhaseCreateApp,
+	PhaseCreateWorkload,
+	PhaseCreateWorklodInstance,
+	PhaseCreateAppVersion,
 	PhaseCompleted,
 	PhaseDeprecated,
```

Comment:
We don't know when a specific phase finished, only that smth finished

Reply:
I don't understand to be honest

Reply:
Here we only emit events when something has been completed. However, it is not clear what phase is completed.

Reply:
this is just name of a phase, has nothing to do with events

Reply:
The comment was created before this function was added: https://github.com/keptn/lifecycle-toolkit/pull/589/files#diff-76b0b0d4fa3d647bf4e029e0cd71a1acaa7fe9ba4245333549252e74bfa46119R54

Now events have all the data I think is necessary to correlate it with other signals! 👍